src/content/structured/components/popover-menu/guidance.mdx

```diff
@@ -0,0 +1,270 @@
+/>
+
+### Menu toggle items
+
+Use the "toggle" variant of menu items for actions that provide an ability to toggle something on or off. Menu toggle items will display a checkmark when toggled on and hide it when toggled off.
+
+Like default menu items, provide an icon or a description to add additional context if required.
+
+<DoDontCaution
+  imageSrc={popoverFig6}
+  imageAlt="A popover menu containing two menu toggle items. Their labels read ‘Show/Hide’ and ‘Lock/Unlock’ and display a tick indicating that they are toggled on."
+  state="good"
+  caption="Use menu toggle items to show a selection state alongside the action."
+/>
+
+### Menu groups
+
+Use menu groups to organise menu items into sections within the popover menu. Provide a label for the menu group to describe its contents. Alternatively, leave the menu group label blank if a description is unnecessary.
+
+<DoDontCaution
+  imageSrc={popoverFig7}
+  imageAlt=" A popover menu with a menu group for ‘Edit options’, a menu group for ‘View options’, an untitled menu group containing unrelated actions, and a final untitled menu group containing destructive actions."
+  state="good"
+  caption="Use menu groups to create sections within a popover menu."
+/>
+
+### Parent menu items
+
+Use a parent menu item to open sub-menus that include more menu items. Use this to contain similar actions or options relating to the menu group name.
+
+<video
+  width="100%"
+  muted
+  controls
+  loop
+  title="An animation showing a popover menu with a parent menu item titled ‘Zoom’. A sub-menu containing menu items for different zoom levels is opened after the ‘zoom’ parent menu item is clicked. After the back button is clicked in the sub-menu the main menu is displayed again."
+>
+  <source src={popoverVid2} type="video/mp4" />
+</video>
+
+### Sub-menus
+
+Use a sub-menu to contain a parent menu item’s children items. Use sub-menus for multiple related actions that are opened from the main menu.
+
+The sub-menu displays the same label as the parent menu item. Clicking the back button in a sub-menu will return the user to the parent menu item in the main menu.
+
+Use menu items, toggle menu items, menu groups or parent menu items in a sub-menu but avoid nesting too many levels of sub-menus as this can be confusing to navigate.
+
+<video
+  width="100%"
+  muted
+  controls
+  loop
+  title="An animation showing a popover menu that contains four parent menu items each with nested parent menu items in their sub-menus. The user clicks ‘help options’ to go into the sub-menu, then clicks ‘information’ to go into a second sub-menu, then clicks ‘tutorials’ to go into a third sub-menu."
+>
+  <source src={popoverVid3} type="video/mp4" />
+</video>
+
+## Accessibility considerations
+
+When specifying the trigger element for a popover menu, make use of ‘aria-expanded’ and ‘aria-collapsed’ so that a screen reader announces the appearance of the popover menu.
```

Comment:
Is this not already built into the component so the consumer is unable to not use the aria attributes?

Reply:
This isn't built in functionality, since we won't be sure what component consumers are using to display the popover, it seemed the best way to make sure they add aria-expanded and aria-collapsed was to add it to the guidance.